src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -85,7 +85,7 @@ class QueryAppsDataValidator {
     if (!HasResponseKey()) {
       return false;
     }
-    return ValidateAppDataAndOsAndLanguagesData();
+    return true;
```

Comment:
@AGritsevich I've read it. I'm just wondering when the reqs will be approved - are you going to return back this portion of code? Where are you going to keep it?